apps/client/src/widgets/type_widgets/options/text_notes/date_time_format.ts

```diff
@@ -0,0 +1,79 @@
+import OptionsWidget from "../options_widget.js"; 
+import { t } from "../../../../services/i18n.js";
+import type { OptionMap } from "@triliumnext/commons"; 
+
+
+const TPL = /*html*/`
+<div class="options-section">
+    <h4>${t("options.customDateTimeFormatTitle", "Custom Date/Time Format (Alt+T)")}</h4>
+
+    <p>
+        ${t("options.customDateTimeFormatDesc1", "Define a custom format for the date and time inserted using the Alt+T shortcut.")}
```

Comment:
kindly check, how these i18n strings are handled across the rest of the project:
usually it's just the key that is used in the widget (e.g. the  `options.customDateTimeFormatTitle` part) and the string is then added to the English i18n file


examples:
https://github.com/TriliumNext/Notes/blob/develop/apps/client/src/widgets/type_widgets/options/text_notes/text_auto_read_only_size.ts
`text_auto_read_only_size.label`

`en/translation.json` file
https://github.com/TriliumNext/Notes/blob/56d4d7c20f775eed730fc10d88d35cb75390af1b/apps/client/src/translations/en/translation.json#L1431